docs/cheatsheet/insert.rst

```diff
@@ -0,0 +1,21 @@
+.. _ref_cheatsheet_insert:
+
+Insert
+======
+
+Insert basic movie stub:
+
+.. code-block:: edgeql
+
+    INSERT Movie {
+        title := 'Dune',
+        year := 2020,
+        directors := (
+            SELECT Person
+            FILTER
+                .last_name = 'Villeneuve'
+            # the LIMIT is needed to satisfy the single
+            # link requirement validation
+            LIMIT 1
+        )
+    }
```

Comment:
Need to show a nested insert example.

Reply:
Aren't we in the process of redefining nested `INSERT` syntax? I'd like to postpone an example until the time when it's finalized.

Reply:
We are, but I didn't mean showing a nested insert for a tree structure (which can be quite complex due to DETACHED). Just a A { b := B } kinda insert would suffice.

Reply:
But my understanding was that one of the options we're possibly re-introducing is the nice nested `INSERT` syntax like:
```
INSERT Foo {
    a := 'hello',
    b: {
        val := 111,
    },
};
```
or slightly more verbose like:
```
INSERT Foo {
    a := 'hello',
    b: Bar {
        val := 111,
    },
};
```

Reply:
Yes, when we introduce the new syntax we'll update the cheat sheet. People need to know how to do it **now**.

Reply:
Example added.

Reply:
Cool, tx